src/views/Pools/components/PoolCard.tsx

```diff
@@ -128,7 +127,7 @@ const PoolCard: React.FC<HarvestProps> = ({ pool }) => {
         </CardTitle>
         <div style={{ marginBottom: '8px', display: 'flex', alignItems: 'center' }}>
           <div style={{ flex: 1 }}>
-            <Image src={`/images/tokens/${image || tokenName}.png`} width={64} height={64} alt={tokenName} />
+            <Image src={`/images/pools/${poolImage}.svg`} alt={tokenName} width={64} height={64} />
```

Comment:
I created a pools image folder because we need the tokens folder with original token images for the `add token to MetaMask` feature.